setup.cfg

```diff
@@ -92,6 +92,26 @@ runtime =
     Werkzeug>=3.0.0
     xmltodict>=0.13.0
 
+# required to actually run localstack on the host
+s3 =
+    awscrt>=0.13.14
+    boto3>=1.26.121
+    botocore>=1.31.2
+    cbor2>=5.2.0
+    dnspython>=1.16.0
+    docker==6.0.1
+    jsonpatch>=1.24
+    hypercorn>=0.14.4
+    localstack-client>=2.0
+    pyopenssl>=23.0.0
+    readerwriterlock>=1.0.7
+    requests-aws4auth>=1.0
+    Werkzeug>=3.0.0
+    xmltodict>=0.13.0
+    # TODO: remove those 2 dependencies
+    flask>=3.0.0
+    Quart>=0.19.2
```

Comment:
only thing is that we're going to re-introduce the hard dependency on `Flask`, because in the default `runtime` we were counting on `Moto` to install it for us. But it's planned to remove, we can do it explicitly then?

Reply:
I like the new naming (`base-runtime`), so this conversation is resolved.